.github/dependabot.yml

```diff
@@ -12,6 +12,15 @@ updates:
       action-dependencies:
         patterns:
           - "*"
+        update-types:
+          - "major"
+          - "minor"
+      action-patch-updates:
+        update-types:
+          - "patch"
+        schedule:
+          interval: cron
+          cronjob: "0 0 1,15 * *"
```

Comment:
There isn't exactly a way to specify "twice a month", so instead this runs on the 1st and 15th of the month.